asn1crypto/algos.py

```diff
@@ -371,6 +378,8 @@ def hash_algo(self):
             'sha256_ecdsa': 'sha256',
             'sha384_ecdsa': 'sha384',
             'sha512_ecdsa': 'sha512',
+            'ed25519': 'sha512',
+            'ed448': 'shake256',
```

Comment:
Late to the party - I wonder if those values are correct. Those hashes are used deeply internally and should not be passed to as a parameter to signature generation/verification functions.

I have proposed setting "raw" value here in https://github.com/wbond/asn1crypto/pull/230 - up for discussion!